webscraper.js

```diff
@@ -9,22 +9,26 @@ build();
 if (require.main === module) {
   //Get folders
   const dirs = p => readdirSync(p).filter(f => statSync(p+"/"+f).isDirectory());
-  const folders = dirs(join(dirname(__dirname), 'product_classification_test_data'));
+  const folders = dirs(join(dirname(__dirname), 'fathom-products' , 'product_classification_test_data'));
+  driver.manage().window().maximize();
+  driver.manage().timeouts().setScriptTimeout(100000); // 10 seconds
 
-  for(var i = 0; i < folders.length; i++){
-      var currUrl = join('file://', dirname(__dirname), 'product_classification_test_data', folders[i], 'source.webarchive');
-      var filepath = join(dirname(__dirname), 'product_classification_test_data', folders[i], 'nodes.txt');
+
+  folders.forEach(function(item){
+      var currUrl = join('file://', dirname(__dirname), 'fathom-products' , 'product_classification_test_data', item, 'source.webarchive');
 
       //Get the page
       driver.get(currUrl);
 
       //Get dimensions and css data
-      driver.executeScript(function() {
-        var doc_dimensions = {};
+      driver.executeAsyncScript(function(dir) {
+        var callback = arguments[ arguments.length - 1 ];
+        var src = document.documentElement.outerHTML;
+        var node_properties = {};
         var all = document.getElementsByTagName("*");
         for (var j = 0; j < all.length; j++) {
              var curr = all[j].getBoundingClientRect();
-             doc_dimensions[j.toString()] = {
+             node_properties[j.toString()] = {
```

Comment:
I don't think you have to call toString() here; JS will happily mash it down to a string itself.